benchmark-service/benchmark/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7-slim-stretch
```

Comment:
Sorry one last comment. Let's move this up a directory. It shouldn't be a part of the Python package.